src/angular-app/languageforge/lexicon/shared/share-with-others/user-management.component.ts

```diff
@@ -58,6 +67,31 @@ export class UserManagementController implements angular.IController {
     });
   }
 
+  onOwnershipTransfer($event: {target: Partial<User>}) {
+    if($event.target.isInvitee){
+      const attemptedToTransferToAnInviteeMessage = 'Please retry after the invitee opens the project.';
+      this.modal.showModalSimple('Ownership was not transferred', attemptedToTransferToAnInviteeMessage, 'OK', 'Close').then(() => {
+      })
+    }
+    else{
```

Comment:
If I'm reading the conditional logic right, we should never hit this code since we don't show the `make owner` option for invitees.  Can you explain how we would get here?

And even then, I don't like having a modal that pops up to tell the user they cannot do something.  We should just not have the option all together in that case.

Reply:
I suggest deleting this block.

Reply:
deleted in latest commit